dask/array/core.py

```diff
@@ -3493,6 +3498,72 @@ def _vindex(x, *indexes):
     return result_1d.reshape(broadcast_shape + result_1d.shape[1:])
 
 
+def _vindex_dask_arrays(x, *indices):
+    """Point wise indexing with Dask Arrays"""
+
+    from .creation import arange
+
+    x = asarray(x)
+
+    oth_inds = []
+    arr_inds = []
```

Comment:
Are these short for other and array or something else?  If so I recommend adding the extra two characters here. I think that the cost of extra typing here does not outweigh the loss of information.

Reply:
Yes. Was trying to keep line length short later on. That said, no strong attachment to this or another variable name.